client/src/app/shared/services/ingest.service.spec.ts

```diff
@@ -84,6 +84,46 @@ describe('Ingest Service', () => {
     ['files', 'protocols', 'biomaterials', 'processes', 'projects'].forEach(makeTest);
   });
 
+  describe('project entity functions', () => {
```

Comment:
Given you've just used get, we don't need these. Shall we get rid of them? They might be useful in the future but also easy to find the commit/rewrite again

Reply:
I'm sorry, I didn't see the functions you created :( Yeah let's remove this for now. I'll think more about how to design that data-table component.